src/VisualStudio/Core/Def/Implementation/AbstractEditorFactory.cs

```diff
@@ -58,6 +58,29 @@ public void SetEncoding(bool value)
         int IVsEditorFactory.Close()
             => VSConstants.S_OK;
 
+        public int ChooseEditorFactory(
+            string pszMkDocument,
+            IVsHierarchy pHier,
+            uint itemid,
+            IntPtr punkDocDataExisting,
+            ref Guid rguidLogicalView,
+            out Guid pguidEditorTypeActual,
+            out Guid pguidLogicalViewActual)
+        {
+            if (rguidLogicalView == VSConstants.LOGVIEWID.Designer_guid)
```

Comment:
Does this allow us to delete any of the stuff that's in the other parts of this editor factory? (like, say the CreateWinFormsEditorInstance method)